src/patternfly/components/Table/table.scss

```diff
@@ -954,39 +924,35 @@
 
 // Check table cell
 .#{$table}__check {
-  --#{$table}--cell--FontSize: var(--#{$table}__check--input--FontSize);
+  --#{$table}--cell--PaddingLeft: var(--#{$table}__check--PaddingLeft);
+  --#{$table}--cell--PaddingRight: var(--#{$table}__check--PaddingRight);
 
-  label:not([disabled]),
-  input:not([disabled]) {
-    cursor: pointer;
-  }
-
-  label {
+  .#{$check}.pf-m-standalone {
     display: inline-flex;
-    padding: inherit;
-    margin-block-start: calc(var(--#{$table}--cell--PaddingTop) * -1);
-    margin-block-end: calc(var(--#{$table}--cell--PaddingBottom) * -1);
-    margin-inline-start: calc(var(--#{$table}--cell--PaddingLeft) * -1);
-    margin-inline-end: calc(var(--#{$table}--cell--PaddingRight) * -1);
 
+    thead & {
+      vertical-align: bottom;
+    }
+
+    tbody & {
+      vertical-align: top;
+    }
   }
 
-  @at-root .#{$table}__thead .#{$check}.pf-m-standalone {
-    --#{$check}__input--TranslateY: var(--#{$table}__thead__check--input--TranslateY);
+  // TODO: remove label wrapper at breaking change
+  label {
+    display: contents;
```

Comment:
I wonder if this will cause an a11y issue since last I read, display: contents still isn't a great thing for elements with semantic value specifically around remaining in the a11y tree properly.